core/src/events/event_handler.rs

```diff
@@ -16,7 +16,7 @@ pub trait ViewHandler: Any {
 
 impl dyn ViewHandler {
     /// Check if a view handler is a certain type.
-    pub fn is<T: ViewHandler + 'static>(&self) -> bool {
+    pub fn is<T: Any + 'static>(&self) -> bool {
```

Comment:
I'm pretty sure you can remove the `'static` constraint, because `Any` already implies it.

```suggestion
    pub fn is<T: Any>(&self) -> bool {
```